api/client.yaml

```diff
@@ -1589,6 +1831,33 @@ components:
           description: Customers name suffix. "Jr", "PH.D."
         type:
           $ref: '#/components/schemas/CustomerType'
+        businessName:
+          type: string
+          description: Business Name for business type customers
+          example: Alphabet Inc.
+        doingBusinessAs:
+          type: string
+          description: Doing Business As (DBA) name for business type customers
+          example: Google
+        businessType:
+          type: string
+          description: Business Type for business type customers - "Individual, sole proprietor, or single-member LLC", "Corporation", "Nonprofit organization", "Partnership", "Limited Liability Company"
```

Comment:
Can we format this as a multi-line string? The current format is pretty hard to read.

Reply:
I added a new BusinessType schema as an enum instead and removed the types in this long description